user/user_lego_sensor.c

```diff
@@ -63,10 +63,20 @@ static ssize_t address_show(struct device *dev, struct device_attribute *attr,
 	return snprintf(buf, LEGO_NAME_SIZE, "%s\n", sensor->sensor.address);
 }
 
+static ssize_t text_value_store(struct device *dev, struct device_attribute *attr,
+			        const char *buf, size_t count)
+{
+ 	struct user_lego_sensor_device *sensor = to_user_lego_sensor_device(dev);
+ 
+	return snprintf( sensor->text_value, USER_LEGO_SENSOR_TEXT_VALUE_SIZE, buf );
```

Comment:
How is this even working? There's no format specifier!? I'll fix and add to the PR. I swear I tested it.